src/libstd/sys_common/backtrace.rs

```diff
@@ -101,59 +103,35 @@ fn filter_frames(frames: &[Frame],
         return (0, 0);
     }
 
-    // We want to filter out frames with some prefixes
-    // from both top and bottom of the call stack.
+    // Frames to remove from the top of the backtrace.
+    //
+    // The raw form is used so that we don't have to demangle the symbol names.
+    // The `a::b::c` form can show up on Windows/MSVC.
     static BAD_PREFIXES_TOP: &'static [&'static str] = &[
```

Comment:
Can we remove this array entirely? Currently `rust_panic` is the "canonical frame name" of the panic, and if there's a lower frame we should start from then we should just give that a canonical name instead.